src/components/SummaryCard.tsx

```diff
@@ -1,10 +1,34 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import icon from '../assets/icon-music.svg';
+import PricePlan from './PricePlan';
+
 import hero from '../assets/illustration-hero.svg';
 
+type PricePlanType = {
```

Comment:
Types could go under src/types

Reply:
I've added a new PriceType alias in src/types.ts file